recipe/meta.yaml

```diff
@@ -19,8 +19,8 @@ requirements:
     - pip
     - python
```

Comment:
@CameronTStark maybe it's as simple as changing this to `python >=3.5`? Might need to modify the `run` section below too.

Reply:
That was it, thanks!